packages/lesswrong/components/common/KarmaDisplay.tsx

```diff
@@ -14,13 +20,27 @@ const KarmaDisplay = ({document, placement="left"}: {
     ? document.afBaseScore
     : null;
   const {LWTooltip} = Components;
+
+  const commentId = !!annualReviewMarketCommentId ? annualReviewMarketCommentId : undefined
+
+  const {document: comment} = useSingle({
+    documentId: commentId,
+    collectionName: "Comments",
+    fragmentName: "CommentsListWithParentMetadata",
+  })
```

Comment:
This should probably have `skip: !annualReviewMarketCommentId` so that it doesn't query for an arbitrary comment (which is I _think_ what happens if you query a collection while not passing in any documentId?)